### PR TITLE
Add grape-jdk benchmark apps

### LINKED ISSUE #1018: Add java benchmark apps for GAE engine
We need some benchmark apps for `grape-jdk`, maybe sample implementation for famous graph algorithms like `sssp`, `pagerank` and `wcc`, e.t.c.  These sample apps can be a vivid illustration of `grape-jdk` usage.

### PR DESCRIPTION
**Pull request by zhanglei1949**
## What do these changes do?

- Add implementation for java benchmarking apps
- Java app benchmarking entrance.
- Refactor of java driver apps.

## Related issue number

Resolves #1018

### CODE REVIEW

File: analytical_engine/java/grape-jdk/src/main/java/com/alibaba/graphscope/fragment/ArrowFragment.java [hunk 1]
```diff
@@ -229,6 +229,9 @@ boolean isOuterVertex(
     <DATA_T> VertexDataColumn<DATA_T> vertexDataColumn(
             int vertexLabelId, int propertyId, @FFISkip DATA_T unused);
 
+    // // @CXXTemplate(cxx = {"uint64_t"}, java = {"Long"})
+    // // @CXXTemplate(cxx = {"double"}, java = {"Double"})
+    // // @CXXTemplate(cxx = {"uint32_t"}, java = {"Integer"})
     // @FFINameAlias("GetData")
```
Comment (sighingnow) [inline analytical_engine/java/grape-jdk/src/main/java/com/alibaba/graphscope/fragment/ArrowFragment.java:235]: I suggestion to delete those (seems) unused comments.

It looks like this pull request has cleaned many such stuffs, and add some similar things (might be cleaned in further PRs again) at the same time.

File: analytical_engine/java/grape-runtime/src/main/java/com/alibaba/graphscope/parallel/ParallelPropertyMessageManager.java [hunk 2]
```diff
@@ -0,0 +1,266 @@
+package com.alibaba.graphscope.parallel;
```
Comment (sighingnow) [inline analytical_engine/java/grape-runtime/src/main/java/com/alibaba/graphscope/parallel/ParallelPropertyMessageManager.java:1]: Add copyright header please.

Please double check all of other java files under `analytical_engine/java` as well.

File: analytical_engine/java/grape-runtime/src/main/java/com/alibaba/graphscope/annotation/sample/AnnotationInvoker.java [hunk 3]
```diff
@@ -123,12 +127,31 @@
...
+            //                                name = "edgeDataColumn",
+            //                                parameterTypes = {"DATA_T"},
+            //                                returnType =
+            // "com.alibaba.graphscope.ds.EdgeDataColumn<DATA_T>",
+            //                                templates = {
+            //                                    @CXXTemplate(cxx = "int32_t", java = "Integer"),
+            //                                    @CXXTemplate(cxx = "int64_t", java = "Long"),
+            //                                    @CXXTemplate(cxx = "double", java = "Double")
+            //                                })
+            //                    }),
```
Comment (sighingnow) [inline analytical_engine/java/grape-runtime/src/main/java/com/alibaba/graphscope/annotation/sample/AnnotationInvoker.java:154]: Delete the temporary comments, if it is not used anymore.

**Review by sighingnow — APPROVED** [review]
LGTM. Just some minor comments.

**Comment by sighingnow** [comment]
You may need to add the benchmark numbers (if there is any) to the `analytical_engine/java/README.md` as well. I think it would be better.

**Comment by zhanglei1949** [comment]
> You may need to add the benchmark numbers (if there is any) to the `analytical_engine/java/README.md` as well. I think it would be better.

OK, I will add the benchmarking results in this pr.

**Comment by sighingnow** [comment]
@zhanglei1949 please avoid merging if CI still fail, the main branch will be broken and will block other developers who want to test and merge their PRs... especially when we have no idea about why it keeps failing and rerun doesn't resolve the issue...

**Comment by zhanglei1949** [comment]
> @zhanglei1949 please avoid merging if CI still fail, the main branch will be broken and will block other developers who want to test and merge their PRs... especially when we have no idea about why it keeps failing and rerun doesn't resolve the issue...

Sorry for the ci problem, will be more cautious.

**Pull request merged**